README.md

```diff
@@ -1 +1,45 @@
-# node-modules-json-lab
\ No newline at end of file
+# Importing JSON Files Lab
+
+## Objectives
+
+1. Implement a JSON config file
```

Comment:
I'm not sure "implement" is the right word here. Really, the third objective sums it all up -- "Use an external JSON configuration file in a script"